tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -4741,3 +4743,46 @@ def test_timestamp_different_from_params(self):
             response = self.do_request(query)
 
             assert response.status_code == 400, query_text
+
+    @mock.patch("sentry.search.events.builder.raw_snql_query")
+    def test_removes_unnecessary_default_project_and_transaction_thresholds(self, mock_snql_query):
+        mock_snql_query.side_effect = [{"meta": {}, "data": []}]
+
+        ProjectTransactionThreshold.objects.create(
+            project=self.project,
+            organization=self.organization,
+            # these are the default values that we use
+            threshold=DEFAULT_PROJECT_THRESHOLD,
+            metric=TransactionMetric.DURATION.value,
+        )
+        ProjectTransactionThresholdOverride.objects.create(
+            transaction="transaction",
+            project=self.project,
+            organization=self.organization,
+            # these are the default values that we use
+            threshold=DEFAULT_PROJECT_THRESHOLD,
+            metric=TransactionMetric.DURATION.value,
```

Comment:
nit: could we add a test where the metric isn't default -> resulting in an additional tuple?